src/timer.rs

```diff
@@ -1,6 +1,7 @@
 use mlua::prelude::{LuaNumber, LuaTable};
 use mlua::{Lua, Result, ToLua, UserData, UserDataFields, UserDataMethods};
 
+#[derive(Copy, Clone)]
 pub struct Timer {
```

Comment:
Timers need to support Copy so we can copy timers into and out of the heal module in the test. Clone is required when enabling Copy. No code changes are required to support this; it's all automatic.